guide/README.md

```diff
@@ -0,0 +1,285 @@
+
+```bash
+csg_fmatch --top ../atomistic/topol.tpr --trj ../atomistic/traj.trr --options fmatch.xml --cg water.xml (--begin $equi ) # ( ) denotes additional option: , --begin # time frame in GROMACS units (ps) to start analysis (e.g. 1 to 5)
+```
+
+To obtain the CG potential, the CG force has to be integrated. (see the appropriate lines in the `run.sh` script):
+
+```bash
+csg_call table integrate CG-CG.force CG-CG.pot # integrates the table
+csg_call table linearop CG-CG.pot CG-CG.pot -1 0 # multiplication of all table values by -1 (potential)
+```
+
+Change the spline grid (`step`), blocksize and parameter constrainedLS. This should get a feeling for the whole procedure.
+
+## Running of CG simulation
+
+To run a CG simulation with GROMACS, the potential has to be converted to a potential table, GROMACS can read (`table_CG_CG.xvg`). (Check the inverse section in the `fmatch.xml` for the corresponding options):
+
+```bash
+csg_call --options fmatch.xml --ia-name CG-CG --ia-type non-bonded convert_potential gromacs --clean input.pot table_CG_CG.xvg # calls convert_potential gromacs. Unsampled regions for distances smaller than the min value are extrapolated.
+```
+
+To run a CG simulation, you will need the `conf.gro`, `topol.top`, `index.ndx` and `grompp.mdp` files. You can use the ones of the ibi tutorial and adjust the number of timesteps and output settings. Then run the simulation. Afterwards, you can calculate the RDF and thermodynamic data as explained in the IBI tutorial. You can also use the `Extract_Energies.sh` script of the ibi tutorial. When calculating the RDF keep in mind that now you don't need a mapping file and the --cg option can be omitted.
+
+Compare the CG potential, the RDF and thermodynamics with the ones of the IBI, IMC or RE method (or any other method) and with the atomistic simulation. You will see that different methods lead to significantly different interaction potentials and a single site water model with a pair interaction potential is not capable to reproduce the RDF and thermodynamics at the same time. (Reason: three-body contributions are important but cannot be projected on a 2-body coarse-grained force-field. The incorporation of non-bonded 3-body interactions is work in progress. An extension to analytic non-bonded 3-body interactions will be released soon).
```

Comment:
"... is not capable to reproduce ..." -> "... is not capable of reproducing ..."